### ISSUE TITLE
How to form the nuimage dataset?

### ISSUE THREAD

**Issue by xuxiaoxxxx** [ADLab-AutoDrive/BEVFusion#16]
How to form the nuimage dataset? Can I form the josn by " python tools/data_converter/nuimage_converter.py " ?

**Comment by azureduck88** [comment]
I have the same question. When I ran the code, it complained "annotations/nuimages_v1.0-train.json" can't be found. My guess is the nuimages dataset was not correctly set up. Once downloading the whole nuimages dataset from its official site, what else steps should we run to correctly set up the dataset? Many thanks!

**Comment by tingtingliangvs** [comment]
Yes, put the nuiamge dataset in the default folder './data/nuimages' or set the '--data-root' argument.
